contracts/vaults/RibbonDeltaVault.sol

```diff
@@ -233,5 +280,46 @@ contract RibbonDeltaVault is RibbonVault, OptionsDeltaVaultStorage {
             counterpartyThetaVault.optionAuctionID(),
             orders
         );
+
+        _updatePPS();
+    }
+
+    /**
+     * @notice Updates the price per share of the current round. The current round
+     * pps will change right after call rollToNextOption as the gnosis auction contract
+     * takes custody of a % of `asset` tokens, and right after we claim the tokens from
+     * the action as we may recieve some of `asset` tokens back alongside the oToken,
+     * depending on the gnosis auction outcome. Finally it will change at the end of the week
```

Comment:
rollToNextOption: calling VaultLifecycle.placeBid gives gnosis contract our funds we approved. pps changes for some time here. 

claimParticipantOrder: every leaf in the if/else tree its possible for some amount of bidding token to be sent back to us. mainly in [this](https://github.com/gnosis/ido-contracts/blob/a766a8b30de14781961158b772393cc49b900714/contracts/EasyAuction.sol#L613) case if settle price comes out to some number where we get some of `asset` back. pps changes for some time here.